From 9d63b7363d63ead3fadf0dccf5c8315e06e1dbcc Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 15 Oct 2018 09:48:31 +0800
Subject: [PATCH] arm64: dts: rockchip: remove io-domain/pmu-io-domain for
 rk1808

RK1808 SoCs support Auto-selection feature, which descript by
PMU_GRF_IO_VSEL1.

Change-Id: I43a240306d95b686b9801d2fe2a2a6f141ec2834
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 18 ------------------
 arch/arm64/boot/dts/rockchip/rk1808.dtsi     | 10 ----------
 2 files changed, 28 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 8cb13a254f9a..f2c7fb0e7b70 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -572,29 +572,11 @@
 	};
 };
 
-&io_domains {
-	status = "okay";
-
-	vccio0-supply = <&vcca_1v8>;
-	vccio1-supply = <&vcca_1v8>;
-	vccio2-supply = <&vcca_1v8>;
-	vccio3-supply = <&vcca_1v8>;
-	vccio4-supply = <&vcca_1v8>;
-	vccio5-supply = <&vccio_3v3>;
-};
-
 &npu {
 	npu-supply = <&vdd_npu>;
 	status = "okay";
 };
 
-&pmu_io_domains {
-	status = "okay";
-
-	pmuio1-supply = <&vcc_1v8>;
-	pmuio2-supply = <&vcc_3v3>;
-};
-
 &pwm1 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 36fd188becb7..30d3cfc717c4 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -168,11 +168,6 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		io_domains: io-domains {
-			compatible = "rockchip,rk1808-io-voltage-domain";
-			status = "disabled";
-		};
-
 		npu_pvtm: npu-pvtm {
 			compatible = "rockchip,rk1808-npu-pvtm";
 			clocks = <&cru SCLK_PVTM_NPU>;
@@ -242,11 +237,6 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		pmu_io_domains: io-domains {
-			compatible = "rockchip,rk1808-pmu-io-voltage-domain";
-			status = "disabled";
-		};
-
 		pmu_pvtm: pmu-pvtm {
 			compatible = "rockchip,rk1808-pmu-pvtm";
 			clocks = <&cru SCLK_PVTM_PMU>;
-- 
2.35.3

